tinygrad/tensor.py

```diff
@@ -54,11 +54,8 @@ def require_init_gpu():
 
 class Tensor:
   did_float_warning = False
-  default_gpu = False
 
-  def __init__(self, data, gpu=None):
-    if gpu is None:
```

Comment:
This is used by train efficientnet

Reply:
Okay, I am not using train efficientnet until now. Maybe adopt to new interface, because None and False are used inconsistently...

Reply:
None uses default while False uses CPU
https://github.com/geohot/tinygrad/blob/master/tinygrad/tensor.py#L60

Reply:
This is interesting, I didn't notice that, sorry. Give me some time, I'll try to fix this issue.

Reply:
but that reminds me that this breaks `.cpu()` #128 should fix it